tests/Endpoints/TasksTest.php

```diff
@@ -82,27 +83,30 @@ public function testGetOneTaskIndex(): void
         $this->assertIsArray($response['details']);
     }
 
-    public function testGetAllTasksIndex(): void
+    public function testGetAllTasksByIndex(): void
     {
         $response = $this->index->getTasks();
-        $preCount = \count($response['results']);
-        [$promise, $response] = $this->seedIndex();
+        $firstIndex = $response['results'][0]['uid'];
+
+        $newIndex = $this->createEmptyIndex('a_new_index');
+        $newIndex->updateDocuments(self::DOCUMENTS);
 
-        $this->assertIsArray($promise);
         $response = $this->index->getTasks();
-        $this->assertCount($preCount + 1, $response['results']);
```

Comment:
The test is basically the same, but now we cannot trust the size of the list anymore, so the idea is to just compare the most recent item if they changed, we can ensure that we are filtering by index. And in the other similar test the opposite is what we look for.